tests/test_bounding_box.py

```diff
@@ -4,10 +4,11 @@
 import numpy
 import pytest
 
-import unittest.mock as mock
+# noinspection PyUnresolvedReferences
+import mock  # move defined in ``tests`` top module
```

Comment:
This is a regression from the line that was removed (e.g. keep the L7 that was seemingly removed above).

Reply:
corrected in latest commit